doc/support_multiple_sql_statements.md

```diff
@@ -0,0 +1,56 @@
+# _Design:_ Support multiple SQL statements
+
+## Overview
+
+This is a design doc on supporting multiple SQL statements in one magic command cell.
+
+## Problem
+
+Currently our Jupyter magic command `%%sqlflow` only supports to run one statement a time. If a user has multiple SQL statements to run, he/she needs to type `%%sqlflow` many times, which is not ideal.
+
+## Design Choice
+
+There are several design choices to make.
+
+### Splitting Location: Client-side vs Server-side
+
+While splitting at the client-side is relatively simple to implement. We prefer to split at the server-side for the following reasons.
+Loose coupling. SQLFlow server can use its lexer/parser to split SQL statements accurately. 
+Extensibility. SQLFlow server can run a sequence of SQL statements to form a workflow.
```

Comment:
More details about what workflow is? We would like to analyze the dependencies of the execution order of all the SQL statements and form an execution graph automatically, then parallelly execute the graph nodes.

Reply:
Sure. Added.